Library/src/Multiplayer/MultiplayerConnection.cpp

```diff
@@ -632,7 +632,14 @@ ConversationSystem* MultiplayerConnection::GetConversationSystem() const
 
 ConnectionState MultiplayerConnection::GetConnectionState() const
 {
-	return static_cast<ConnectionState>(Connection->GetConnectionState());
+	if (Connection != nullptr)
+	{
+		return static_cast<ConnectionState>(Connection->GetConnectionState());
```

Comment:
If my understanding is correct, this will not create a multiplayer connection for returning users. How will this affect a user opening and logging in to a client application with a non-expired, valid authentication or refresh token from a previous login?

Using this API:
<img width="1081" alt="Screenshot 2024-03-21 at 08 47 59" src="https://github.com/magnopus-opensource/connected-spaces-platform/assets/74913022/71ea3f9c-4409-45df-b821-bc63b92f9868">

Reply:
Similar question for the Exchange Key login as well:
<img width="758" alt="Screenshot 2024-03-21 at 09 00 13" src="https://github.com/magnopus-opensource/connected-spaces-platform/assets/74913022/2a1aa782-c512-4a5f-96fc-080fea170250">

Reply:
I think there could be an issue as things stand. For example on web in case where:
1. User logs in to OKO
2. User closes the tab
3. User opens OKO in another tab

At step 3, currently the web client will restore the session using `refreshSession` with token information the web client previously saved to local storage. There won't be any call to login.

Is it the case that no multiplayer connection would be created in this scenario, unless login is called?